stdlib/public/SDK/Foundation/NSGeometry.swift

```diff
@@ -20,35 +20,15 @@ import CoreGraphics
 // NSRectEdge
 //===----------------------------------------------------------------------===//
 
-// In the SDK, the following NS*Edge constants are defined as macros for the
-// corresponding CGRectEdge enumerators.  Thus, in the SDK, NS*Edge constants
-// have CGRectEdge type.  This is not correct for Swift (as there is no
-// implicit conversion to NSRectEdge).
-
-@available(*, unavailable, renamed: "NSRectEdge.MinX")
-public var NSMinXEdge: NSRectEdge {
```

Comment:
Foundation's `NSGeometry.h` already has these unavailable messages on the relevant types directly for all platforms except for 32-bit Mac. (There we define the `NS` types in terms of `CG` types, except that 32-bit Mac gets its own definitions.)